src/__experimental__/components/form-field/form-field.component.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import FormFieldStyle from './form-field.style';
+import Label from '../label';
+import FieldHelp from '../field-help';
+
+/**
+ * To import FormField:
+ *
+ *   import FormField from 'carbon-react/lib/__experimental__/components/form-field';
+ *
+ * To render FieldHelp:
+ *
+ *   <FormField label='Name'>
+ *     <input />
+ *   </FormField>
+ */
+const FormField = ({
+  children,
+  fieldHelp,
+  label,
+  labelAlign,
+  labelHelp,
+  labelInline,
+  labelWidth,
+  size
+}) => (
+  <FormFieldStyle>
+    { label && (
+      <Label
+        align={ labelAlign }
+        help={ labelHelp }
+        inline={ labelInline }
+        inputSize={ size }
+        labelWidth={ labelWidth }
+      >
+        { label }
+      </Label>
+    ) }
+
+    { children }
+
+    { fieldHelp && (
+      <FieldHelp labelInline={ labelInline } labelWidth={ labelWidth }>
+        { fieldHelp }
+      </FieldHelp>
+    ) }
+  </FormFieldStyle>
+);
+
+FormField.defaultProps = {
+  size: 'medium'
+};
+
+FormField.propTypes = {
+  children: PropTypes.node,
+  fieldHelp: PropTypes.node,
+  label: PropTypes.node,
+  labelAlign: PropTypes.oneOf(['left', 'right']),
```

Comment:
Could use the `OptionHelper` here?

Reply:
Yeah I will look into those. TBH I think we can do better with the PropTypes, in terms of sharing them across components that need them. For example Textbox does not have many props, but it is composed of many components that have other props, maybe those should be shared and added to the Textbox PropTypes.